.github/workflows/codeql.yml

```diff
@@ -35,3 +35,15 @@ jobs:
         uses: github/codeql-action/autobuild@v2
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v2
+
```

Comment:
Please just remove newlines on 38 and 44 to stay consistent with the other steps. Other than that LGTM!

Reply:
Change has been made!